stylesheets/commons/Miscellaneous.less

```diff
@@ -2266,22 +2266,40 @@ Author(s): trev
 Template: General Collapsible
 Author: warnull
 *******************************************************************************/
-.general-collapsible-default-toggle {
-	display: inline-block;
-	min-width: 48px;
-	text-align: right;
-}
-
 .collapsed .general-collapsible-default-toggle > .general-collapsible-collapse-button,
 .general-collapsible:not( .collapsed ) .general-collapsible-default-toggle > .general-collapsible-expand-button {
 	display: none;
 }
 
+/* Can be removed when GTL is standardized */
 .general-collapsible-default-placeholder {
 	display: inline-block;
 	min-width: 48px;
 }
 
+.general-collapsible-default-header {
+	display: grid;
+	align-items: center;
+	grid-template-columns: 1fr auto 1fr;
+	gap: 1em;
+	padding: 0 0.5rem;
+}
+
+.general-collapsible-default-title {
+	grid-column: 2;
+	white-space: nowrap;
+	overflow: hidden;
```

Comment:
fwiw not a super fan of just cutting longer titles of
but don't see a better solution